.github/ISSUE_TEMPLATE/developer-template.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Developer Template
```

Comment:
What is this template about?

Reply:
Nothing, I'll just delete this, thought it would be better for regular contributors of the repo